README.md

```diff
@@ -76,6 +76,27 @@ To compile a CSV run:
 make export-csv
 ```
 
+### Exporting CSV
```

Comment:
This is specific for GitLab, and should IMHO be part of the toolchain - not this repo. However, as this file already contains build instructions there's not that much downside